package.json

```diff
@@ -119,7 +119,7 @@
     "prettier": "^3.4.2",
     "react-refresh": "^0.16.0",
     "remark-gfm": "^4.0.0",
-    "rollup": "^4.32.1",
+    "rollup": "^2.79.2",
     "rollup-plugin-node-externals": "^8.0.0",
```

Comment:
question: Do we need to restore the versions for the rollup related dependencies as well? https://github.com/Khan/wonder-blocks/pull/2453/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L119-L121

Reply:
In theory, rolling back `rollup` should be good enough. I tested this change manually and it seemed to work just fine.

Reply:
Thanks for confirming!